Philips.CodeAnalysis.Test/Moq/MockObjectsMustCallExistingConstructorsAnalyzerTest.cs

```diff
@@ -496,6 +496,40 @@ public static void Method()
 			}
 		}
 
+		[DataRow("", false)]
+		[DataRow("MockBehavior.Default", false)]
+		[DataRow("1, 2", true)]
+		[DataTestMethod]
+		[TestCategory(TestDefinitions.UnitTests)]
+		public async Task ConstructorsMustExistViaNewMockInferredAsync(string arguments, bool isError)
```

Comment:
My colleague states that we should not end Test method names with "Async", so we have been removing them as we come across them. Thoughts?